src/lib/Microsoft.Health.Common/Storage/BlobContainerClientFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Azure.Identity;
+using Azure.Storage.Blobs;
+using EnsureThat;
+using Microsoft.Health.Common.Auth;
+
+namespace Microsoft.Health.Common.Storage
+{
+    public static class BlobContainerClientFactory
```

Comment:
>static [](start = 11, length = 6)

Is there a reason for static? Normally it is best to have a class/interface so we can have dependency injection.  Overall this is a great improvement though.  Thank you. If this can be made non-static I am fine doing that in a separate PR.  I made a note and will create a work item unless there is a specific reason for the static.

Reply:
thanks - updated to a normal non-static class